### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by paulvi** [obhi-d/eclipse-ep-ide#1]
Is it for C++ only?

Why new View and not extending ProjectExplorer?

please add README.md

**Comment by obhi-d** [comment]
No it is not C++ only.
It is a new View which directly uses/filters the same resources that a default CNF View could display.
I intend to develop upon this new View and add more features, hence the new View. 
While developing I did face some issues while extending the ProjectExplorer, I dont remember what exactly(because this was a side project I did for my CDT, I primarily work in C++).
However if I recall correctly, FilteredTree uses its own (sub-classed) TreeViewer whereas CNF uses its own CommonViewer. There was no way to add the FilteredTree composite to the CNF view (in ProjectExplorer) without some customization. May be I did not try.

I will add a Readme.
